### PR TITLE
refactor: Layers

### PR DESCRIPTION
**Pull request by fjozsef**
@NejcZdovc please review!

### CODE REVIEW

File: src/components/layers/layers.component.ts [hunk 1]
```diff
@@ -9,43 +7,21 @@ import {
...
-            this.renderer.setElementClass(this.buttons[this.current].nativeElement, 'btn-primary', true);
-            this.selectedLayerIndex = 0;
-        }
+    constructor() {
+        this.buttons = ['Base', 'Mod', 'Fn', 'Mouse'];
+        this.current = 0;
     }
 
     selectLayer(index: number) {
-        if (index === this.selectedLayerIndex) {
```
Comment (NejcZdovc) [inline src/components/layers/layers.component.ts]: Please add this back, so that if you click on current button, you don't trigger the whole tree.

**Comment by NejcZdovc** [comment]
Great refactor, this was on my to-do, because this code really needed it.

File: src/components/layers/layers.component.ts (hunk 1 above)
Comment (fjozsef) [inline src/components/layers/layers.component.ts]: Done

**Comment by fjozsef** [comment]
Yes, I don't understand how could I not notice earlier.

**Pull request merged**